tests/integration/test_show.py

```diff
@@ -59,14 +59,23 @@ def test_wrong_input(c):
         c.sql(f'SHOW COLUMNS FROM "{c.schema_name}"."table"')
     with pytest.raises(AttributeError):
         c.sql('SHOW TABLES FROM "wrong"')
+    with pytest.raises(RuntimeError):
+        c.sql(f'SHOW TABLES FROM "wrong"."{c.schema_name}"')
```

Comment:
Is it also possible to add a test where we're explicitly selecting the default catalog name and it works as expected?